docs/howto/garbage-collection.md

```diff
@@ -449,20 +449,61 @@ The uncommitted GC will not clean:
 1. lakeFS server version must be at least [v0.87.0](https://github.com/treeverse/lakeFS/releases/tag/v0.87.0). 
 If your version is lower, you should first upgrade.
 2. Read the [limitations](#limitations) section.
+3. Use [rclone](https://rclone.org/) for backup and restore
+
+
+Running uncommitted GC performed by the following steps:
+
+1. Mark the files to deleted - summary and report will be generated under '<repository storage namespace>/_lakefs/retention/gc/uncommitted/<mark id>/'.
+   The mark id sortable time based ID, each run output into a new one.
```

Comment:
The sentence is not very clear. do you mean that each UGC run generates a new time-based mark id?